code/modules/admin/verbs/adminhelp.dm

```diff
@@ -509,6 +517,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	message_admins(msg)
 	log_admin_private(msg)
 	AddInteraction("Rejected by [key_name].")
+	add_player_interaction("Rejected by [key_name_admin(usr, FALSE)].")
```

Comment:
This "by" behavior is new right? players didn't used to see that. (Same can be said for most of this)

Reply:
Yeah, i'll change up these to be more accurate to what it was. This was just a direct port tbh